src/app/redux/AppReducer.js

```diff
@@ -24,7 +24,7 @@ const defaultState = Map({
     user_preferences: Map({
         locale: null,
         nsfwPref: 'warn',
-        theme: 'light',
+        nightmode: false,
```

Comment:
why change this?

Reply:
it was setting an incorrect default state previously; `nightmode` is the correct prop name

Reply:
(my bug)

Reply:
got it